types/index.d.ts

```diff
@@ -0,0 +1,328 @@
+import { ComponentType, Ref, RefForwardingComponent } from 'react';
+import JQuery = require('jquery');
+
+export type Locale =
+  'en' |
+  'ar' |
+  'az' |
+  'ca' |
+  'cs' |
+  'da' |
+  'de' |
+  'el' |
+  'es' |
+  'et' |
+  'fr' |
+  'he' |
+  'it' |
+  'ja' |
+  'ko' |
+  'lv' |
+  'nb' |
+  'nl' |
+  'pl' |
+  'pt' |
+  'ro' |
+  'ru' |
+  'sk' |
+  'sr' |
+  'sv' |
+  'tr' |
+  'uk' |
+  'vi' |
+  'zhTW' |
+  'zh';
+
+export type LocalePluralize = (n: number) => string;
+
+export interface LocaleTranslations {
+  uploading?: string;
+  loadingInfo?: string;
+  errors?: {
+    default?: string;
+    baddata?: string;
```

Comment:
cammelCase?

Reply:
Unfortunately, no. This is how it looks like https://github.com/uploadcare/uploadcare-widget/blob/master/src/locales/en.js